src/Console/Command/FixCommand.php

```diff
@@ -169,12 +169,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
 
             $configFile = $resolver->getConfigFile();
-            $stdErr->writeln(sprintf('Loaded config <comment>%s</comment>%s.', $resolver->getConfig()->getName(), null === $configFile ? '' : ' from "'.$configFile.'"'));
+            $stdErr->writeln(
+                sprintf(
+                    'Loaded config <comment>%s</comment>%s.',
+                    $resolver->getConfig()->getName(),
+                    null === $configFile ? '' : ' from "'.$configFile.'"'
+                ),
+                OutputInterface::VERBOSITY_VERBOSE
+            );
 
             if ($resolver->getUsingCache()) {
                 $cacheFile = $resolver->getCacheFile();
                 if (is_file($cacheFile)) {
-                    $stdErr->writeln(sprintf('Using cache file "%s".', $cacheFile));
+                    $stdErr->writeln(
+                        sprintf('Using cache file "%s".', $cacheFile),
+                        OutputInterface::VERBOSITY_VERBOSE
```

Comment:
again, we wasted time multiple times investigating why app is not checking all the files, while it was simply ignoring them due to cache